pages/entry.jsx

```diff
@@ -66,22 +66,28 @@ export default React.createClass({
   },
   render() {
     let docTitle;
+    let description;
     let justPostedByUserMessage = null;
     let content;
 
     if (this.state.dataLoaded) {
       docTitle = `${this.state.entry.title}`;
       justPostedByUserMessage = this.state.justPostedByUser ? (<h5 className="col-12 text-center">Thanks for submitting!</h5>) : null;
       content = <ProjectCard {...Utility.processEntryData(this.state.entry)} onDetailView={true} />;
+      description = `${this.state.entry.description}`;
     }
 
     if (this.state.noData) {
       docTitle = NO_ENTRY_TITLE;
       content = NO_ENTRY_BLOCK;
+      description = '';
     }
 
     if (docTitle) {
-      docTitle = <Helmet><title>{ docTitle }</title></Helmet>;
+      docTitle = <Helmet>
+      <meta property="og:title" content= "{ `${this.state.entry.title}` }" />
```

Comment:
this deserves a bit of indentation. Also note that `content= "{ `${this.state.entry.title}` }"` doubles up the templating several times. Instead, just use `content={docTitle}` without the quotes, and without the template string syntax, and using the title